aten/src/ATen/native/native_functions.yaml

```diff
@@ -4642,6 +4642,8 @@
   structured_delegate: sum.IntList_out
   device_check: NoCheck   # TensorIterator
   variants: function, method
+  dispatch:
+    NestedTensorCPU: NestedTensor_sum__dim
```

Comment:
very tiny naming nit: why two underscores between `sum` and `dim`? I read this as `sum_` followed by `dim` indicating an in-place op